### PR TITLE
Custom Image Quality functionality - > added new property in Optimize…

### PR DESCRIPTION
**Pull request by pateljan**
Updated the library to support Custom Image Quality feature with Lossy image optimization.
As per specified in kraken documentation below....
https://kraken.io/docs/custom-image-quality.

Model.OptimizeRequestBase.Quality

### CODE REVIEW

**Comment by Kevin-Bronsdijk** [comment]
Thank you for creating the PR. I will merge the changes and create a new release this evening.

**Comment by Kevin-Bronsdijk** [comment]
Hi, you forgot to write an integration test to make sure all is working fine after applying the changes. In addition, the value is optional, but within your implementation, this is not the case. 

This behavior should be visible in the new tests. The old calls without the value should still be woking fine even if not providing a value. Now you render a result within the output which defaults to 0.

File: src/kraken-net/Model/OptimizeRequestBase.cs [hunk 1]
```diff
@@ -48,5 +48,8 @@ public SamplingScheme SamplingScheme
 
         [JsonProperty("dev")]
         public bool Dev { get; set; }
+
+        [JsonProperty("quality")]
+        public int Quality { get; set; }
```
Comment (Kevin-Bronsdijk) [inline src/kraken-net/Model/OptimizeRequestBase.cs]: This should be optional and only included within the JSON message when a value has been provided.

Comment (pateljan) [inline src/kraken-net/Model/OptimizeRequestBase.cs]: -Set Quality as Optional parameter. Parameter will be included in JSON message when value will be Provided.
-Added Integration Test method.
-Test old methods without quality parameter.

**Pull request merged**